Mcu/f051/Src/peripherals.c

```diff
@@ -495,6 +495,43 @@ void initLed()
 }
 #endif
 
+#ifdef USE_NSLEEP
+void initnSleep()
```

Comment:
these are initialised but don't seem to be used? how do you envisage using them?

Reply:
Putting the gate drivers into sleep mode when idle saves about 600mW. Ideally when "disarmed" nSLEEP would be driven low, then when "armed" get driven high.

Reply:
ok, sounds very worthwhile. Needs some comments and some hooks to use it. We'd also need to be careful with timing - we presumably need some delay between when we get a disarm signal and actually put the gate drivers to sleep.

Reply:
At the very least, I need this initializer here for the pin state.